src/main/resources/public/raml/eiffel-intelligence.raml

```diff
@@ -37,7 +37,7 @@ baseUri: /
 /subscriptions: !include resourceTypes/subscription.raml
 /query: !include resourceTypes/query.raml
 /queryAggregatedObject: !include resourceTypes/query-aggregated_object.raml
-/queryMissedNotifications: !include resourceTypes/query-missed_notifications.raml
+/failedNotifications: !include resourceTypes/failed-notifications.raml
```

Comment:
update the endpoint to use spinal-case also

Reply:
done